bootstrap/pkg/utils/k8utils.go

```diff
@@ -125,19 +126,38 @@ func GetObjectKindFromUri(configFile string) (string, error) {
 		return "", fmt.Errorf("config file must be a URI or a path")
 	}
 
-	appDir, err := ioutil.TempDir("", "")
-	if err != nil {
-		return "", fmt.Errorf("Create a temporary directory to copy the file to.")
-	}
-	// Open config file
-	appFile := path.Join(appDir, "tmp.yaml")
-
-	log.Infof("Downloading %v to %v", configFile, appFile)
-	err = gogetter.GetFile(appFile, configFile)
+	// Check if configFile is a remote file.
```

Comment:
Can we refactor the code to make this easier to test?

One thought would be to define a function isRemoteFile and then we could add a unittest to cover the cases
* remote URI
* relative path
* absolute path.

Reply:
done